packages/wonder-blocks-link/components/link.js

```diff
@@ -120,6 +132,7 @@ export default class Link extends React.Component<SharedProps> {
         const {
             onClick,
             href,
+            openInNewTab,
```

Comment:
Flow is mad about me pulling out both `openInNewTab` and `skipClientNav`. Not sure how to handle this?